pkg/restapi/operation/operations.go

```diff
@@ -17,6 +17,8 @@ import (
 	"net/http"
 	"net/url"
 
+	"github.com/trustbloc/hub-auth/pkg/restapi/common/store/cookie"
+
```

Comment:
can this be grouped with `hub-auth` imports below?

Reply:
will be done in #51